### ISSUE TITLE
Take apart the repo

### ISSUE THREAD

**Issue by pecheny** [ssannandeji/Zenject-2019#126]
Hi! I offer to discuss the idea of separating different repos for Zenject itself and Unity project with examples. (In this case Unity example project repo can refer to zenject as a git submodule).
I use Zenject as a projeny package and it would be nice if package be able to point exactly to the repo.
I already use several packages this way:

```
cd SharedPackages
git submodule add %giturl% %packagename%
```

It works fine for me.
I had tried to put symlink to SharedPackages folder and it works in unity but it's too much for git, it cant reslove symlink pointed to symlink:
`error: readlink("SharedPackages/Zenject4"): Function not implemented`
Maby there are several ways to make link on NTFS and i chose wrong one from git' point of view but however need to make link to link seems a little "ducttaping".

**Comment by svermeulen** [comment]
No, I don't think git for windows handles any of the different kinds of links (last time I checked anyway)

I like to keep the samples together because otherwise it's a maintenance headache for me since I would have to use sub modules or something.  Maybe you could fork the repo then delete everything except the source folder?

**Comment by pecheny** [comment]
Thanks. For now i decided just keep local Zenject repo alone and copy required folder manual to SharedPackages after updating. It seems simpler than syncing forked repo. 
However i will investigate to pipeline with submodules on my own packages. If i will found it solid and smooth then i can fork and separate Zenject repos to show the idea an working example at once.
So you will able take a look and decide will separated repos cause headache in your maintenance pipeline. I guess it depends on tools to use for VCS (git client/ide plugin) and how it handles submodules, since local directory structure will be the same.
